gateway/package.json

```diff
@@ -16,7 +16,7 @@
     "start": "nodemon dist/src/index.js",
     "copy-files": "copyfiles -a 'src/**/schema/*.json' 'src/templates/*.yml' 'test/services/data/**/*.*' dist",
     "test:unit": "jest --verbose --forceExit ./test/",
-    "test:cov": "jest --coverage --forceExit ./test/",
+    "test:cov": "jest -w 2 --coverage --forceExit ./test/",
```

Comment:
Is it really a requirement to limit the number of workers to 2?

Reply:
Seems there's a bug when using jest with `--coverage` that make it run out of memory because of how it keep the ts maps. Limiting worker to 2 solves the issue.